UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/RenderPipelines/RoughRefractionFeature.cs

```diff
@@ -54,24 +94,45 @@ public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderin
 	                descriptor.width /= 4;
 	                descriptor.height /= 4;
 	            }
-
+#if UNITY_2023_1_OR_NEWER
+		        RenderingUtils.ReAllocateIfNeeded(ref m_destination, descriptor, FilterMode.Trilinear,
+			        TextureWrapMode.Clamp, name: "_CameraOpaqueTexture");
```

Comment:
put on one line
fix indentation for block

Reply:
f092e7eaf927044320db7324037797072454be86